lib/mayaHydra/hydraExtensions/sceneIndex/mayaHydraSceneIndex.h

```diff
@@ -252,6 +258,9 @@ class MAYAHYDRALIB_API MayaHydraSceneIndex : public HdRetainedSceneIndex, public
     /// Is using an environment variable to tell if we should pass normals to Hydra when using the render item and mesh adapters
     static bool passNormalsToHydra();
 
+    ///Create the default material from the "standardsurface1" maya material or create a fallback material if it cannot be found
```

Comment:
Seen this one in a few places, name of the default material is camelCase `standardSurface1`, not `standardSurface1`.